### PR TITLE
use styled-jsx/css

### PR DESCRIPTION
**Pull request by alexandercbooth**
As discussed in #14 and #15, styles might get too bloated in the components and can separated out to simplify things.

### CODE REVIEW

File: src/common/styles.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+// @flow
+import css from "styled-jsx/css";
+
```
[the remark this answers is not in the document]
Comment (alexandercbooth) [reply src/common/styles.js]: Ahh yep, that definitely makes sense 😅

**Comment by alexandercbooth** [comment]
Hmm, seems to have failed on "Checkout code" 🤔

**Comment by alexandercbooth** [comment]
That's weird, I didn't close this  ¯\\_(ツ)_\/¯

**Comment by alexandercbooth** [comment]
I think something funky is happening since this repo went public while this PR is from my forked, private copy.

**Comment by rgbkrk** [comment]
Whoop! Hopefully easier now. I pulled the trigger on public so that now.sh would hopefully work on forked PRs.

**Pull request closed without merging**